docs/developer/app-store/legacy-plugins/oidc.mdx

```diff
@@ -23,7 +23,7 @@ Go to Configuration -> Plugins -> OpenID Connect and fill in the fields:
 - **User info URL:** The URL which can be used to fetch user details by using an access token (Required for flow: _Saleor as a resource server_).
 - **OAuth logout URL:** The logout URL which Saleor will return for frontend's logout request (Optional for flow: _Saleor as a client for authorization server_).
 - **Audience:** The Oauth resource identifier. If provided, Saleor will define the audience for each authorization request. Used to fetch user permissions from OAuth provider and map it to Saleor's permission (Optional for both integration flows).
-- **Use OAuth scope permissions:** Use OAuth scope permissions to grant a logged-in user access to protected resources. Your OAuth provider needs to have defined Saleor's permission scopes in format \_saleor:<saleor-permission\> (Required for both integration flows).
+- **Use OAuth scope permissions:** Use OAuth scope permissions to grant a logged-in user access to protected resources. Your OAuth provider needs to have defined Saleor's permission scopes in format \_saleor:&lt;saleor-permission\&lt; (Required for both integration flows).
```

Comment:
The backslashes look wrong. It looks like they were meant to escape the lt/gt entities, but they are in random positions now.

Reply:
Fixed